exercises/concept/cars-assemble/.docs/introduction.md

```diff
@@ -3,7 +3,7 @@
 ## Numbers
 
 Go contains basic numeric types that can represent sets of either integer or
-floating-point values. There a different types depending on the size of value
+floating-point values. There are of different types depending on the size of value
```

Comment:
```suggestion
floating-point values. There are different types depending on the size of value
```

Reply:
Ugh, my brain was reading: "they are of different types" 🙂